A01655713 José Luis Macías Vázquez

josemacvz opened the pull request.

josemacvz committed:
A01655713 José Luis Macías Vázquez

josemacvz commented:
A01655713 José Luis Macías Vázquez

rmroman commented:
Bien, 

Si necesitas ayuda, búscame en hora de asesoría y te ayudo con gusto.

Ganaste 1250 HP

josemacvz committed:
Es el proyecto final.

Todo el proyecto final está en el archivo zip listo para pegarlo en pycharm. Pero también subo cada archivo uno por uno en el caso de que no funcione. Saludos profesor.

rmroman commented:
Proyecto final

Rubro | Puntos | Observaciones
-- | -- | --
Uso de Listas | 10/10 | Muy bien
Uso de Archivos | 0/5 | Muy bien
Elementos en pantalla (3 objetos o más) | 10/10 | Muy bien
Interacción con el usuario | 10/10 | Muy bien
Estado del juego (marcador, vidas, etc.)  | 10/10 | Muy bien
Gana/Pierde  | 5/5 | Muy bien
Sonido  | 5/5 | Muy bien
Menú, Jugar, Puntajes  | 10/10 | Muy bien
800x600  | 5/5 | Muy bien
Funcionalidad/Lógica (choques, vidas, movimiento) | 10/30 | No hay más de lo que hicimos en clase.


75/100
